packages/web-runtime/src/components/ApplicationsMenu.vue

```diff
@@ -23,11 +23,11 @@
         <div v-for="(n, nid) in menuItems" :key="`apps-menu-${nid}`" class="uk-width-1-3">
           <a v-if="n.url" key="apps-menu-external-link" :target="n.target" :href="n.url">
             <oc-icon :name="n.iconMaterial" size="xlarge" />
-            <div v-translate>{{ n.title }}</div>
+            <div v-text="$gettext(n.title)" />
```

Comment:
`v-translate` should be supported by vue-gettext. Maybe it's unsupported to use it on `div`s? But this brings me to another issue: a `div` inside an `a` tag is invalid html markup. Could you try if this is visually still the same if you use a span? You could also use a `<translate>` tag, that will result in a span as well.

Reply:
Uff, great eye! Will go with a `<span>` and `display: block;`.

Also, had a discussion with @fschade yesterday and we figured dropping `v-translate` and `<translate>` in favor of `$gettext()` would be benefical for statical code analysis down the road